### PR TITLE
Add server host to config

### PR DESCRIPTION
**Pull request by nirshirion**

### CODE REVIEW

File: constructors/partial/httpserver.go [hunk 1]
```diff
@@ -77,9 +77,10 @@ type httpServerDeps struct {
 //
 func HTTPServerBuilder(deps httpServerDeps) serverInt.GRPCWebServiceBuilder {
 	builder := server.Builder().SetPanicHandler(deps.panicHandler).SetLogger(deps.Logger.Debug)
+	host := deps.Config.Get(confkeys.Host)
```
Comment (talgendler) [inline constructors/partial/httpserver.go:80]: Why not get the String value right away
```golang
host := deps.Config.Get(confkeys.Host).String()
```

File: constructors/partial/httpserver.go [hunk 2]
```diff
@@ -95,10 +96,11 @@ func (deps httpServerDeps) buildExternalAPI(builder serverInt.GRPCWebServiceBuil
 		builder = builder.RegisterGRPCAPIs(deps.GRPCServerAPIs...) // register grpc APIs
 	}
 	// add GRPC Gateway on top and expose on external REST Port
+	host := deps.Config.Get(confkeys.Host)
```
Comment (talgendler) [inline constructors/partial/httpserver.go:99]: Why not get the String value right away
```golang
host := deps.Config.Get(confkeys.Host).String()
```

File: constructors/partial/httpserver.go [hunk 3]
```diff
@@ -119,12 +121,13 @@ func (deps httpServerDeps) buildExternalAPI(builder serverInt.GRPCWebServiceBuil
 func (deps httpServerDeps) buildInternalAPI(builder serverInt.GRPCWebServiceBuilder) serverInt.GRPCWebServiceBuilder {
 	builder = builder.RegisterGRPCAPIs(health.RegisterInternalHealthService) // add internal GRPC health endpoint
 	// Internal
+	host := deps.Config.Get(confkeys.Host)
```
Comment (talgendler) [inline constructors/partial/httpserver.go:124]: Why not get the String value right away
```golang
host := deps.Config.Get(confkeys.Host).String()
```

**Review by talgendler — APPROVED** [review]
You can commit as is or do this little change

```golang
host := deps.Config.Get(confkeys.Host).String()
```

**Comment by nirshirion** [comment]
> You can commit as is or do this little change
> 
> ```go
> host := deps.Config.Get(confkeys.Host).String()
> ```

Done

**Review by talgendler — APPROVED** [review]

**Pull request merged**